TheLastScrub/delta-green-foundry-vtt-system-unofficial
# Special Training

Adds the ability to track and roll special training from the sheet, and a slight restyle of the Skills section of the sheet to go along with it.

**Important Dev Notes:**
- I split out the Custom Skill / Special Training into its own handlebars partial so that it could be repeated across the different actor sheets with more ease.

**Steps To Test:**

1. Checkout this branch on an existing world, if possible.
2. Make sure existing characters open correctly.
3. Add a special training to an actor using the new UI button on the sheet (next to `Add Custom/Typed Skill`).
4. Give it a name and choose a Stat attribute associated with the Training.
5. Repeat steps 3-4 and choose a Skill, then Typed/Custom Skill associated with the training, so there is a Special Training for each.
6. Roll each special training and make sure everything works as intended.
7. Double check that all other roll types still work.
8. Repeat 1-7 for Unnatural and NPC actors
